src/main/java/com/aziz/paymentapp/controller/AccountController.java

```diff
@@ -0,0 +1,26 @@
+package com.aziz.paymentapp.controller;
+
+import com.aziz.paymentapp.model.AccountDto;
+import com.aziz.paymentapp.service.AccountService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("api/accounts")
```

Comment:
you should be versioning your APIs. If if you have a new version then you can switch traffic gradually and test the new version
```suggestion
@RequestMapping("api/v1/accounts")
```

Reply:
done